### DISCUSSION TITLE
Does this MV3 version have any limitations or issues compared to the original SingleFile?

### DISCUSSION THREAD

**Discussion by ghost** [gildas-lormeau/SingleFile-Lite#31] [Q&A] [answered]
I understand that the version that uses Manifest 3 was created for Chromium browsers, since they dropped support for extensions that use Manifest 2.

There are now two versions, the original SingleFile and SingleFile-MV3. However, Firefox also supports Manifest 3.

So, are there any limitations or problems with MV3 compared to the original?

It's more work to maintain support for two different versions, so I think there must be some motivation.

**Comment by gildas-lormeau** [comment, marked as answer]
It took a while for Google to implement the missing APIs in version 3, but in the end, the 2 extensions have the same features. 

Today, I continue to maintain the 2 extensions, but eventually, the aim is to merge all the code into a single version. It's not a very urgent task, and Firefox is missing a few MV3 APIs, so I'll wait a little longer. Maintaining the two source codes isn't too much of a hassle and allows me to do some code proofreading, by side effect.
